### PR TITLE
deb_version - added metadata

### PR DESCRIPTION
**Pull request by iambibhas**
Related to #1697.

### CODE REVIEW

File: lib/DDG/Spice/DebVersion.pm [hunk 1]
```diff
@@ -1,9 +1,18 @@
-# Get the version of a package in various debian repositories
 package DDG::Spice::DebVersion;
```
Comment (javathunderman) [inline lib/DDG/Spice/DebVersion.pm]: If you could put line 1 under the package declaration with the ABSTRACT declaration comment, it should be ok...

Comment (iambibhas) [inline lib/DDG/Spice/DebVersion.pm]: @javathunderman I added that string to the `description` metadata. Should I add it again as ABSTRACT?

Comment (moollaza) [inline lib/DDG/Spice/DebVersion.pm]: It wouldn't hurt to set an ABSTRACT, but we actually use the `description` (or will be soon) when the IA Pages (duck.co/ia) are further along.

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**